src/general/cpuInfo.go

```diff
@@ -47,29 +47,51 @@ func NewCPULoad() *CPULoad {
 	return &CPULoad{}
 }
 
+// ReadCPULoad reads /proc/stat and returns the average load
+func ReadCPULoad() ([10]float64, error) {
+	data, error := os.ReadFile("/proc/stat")
+	if error != nil {
+		return [10]float64{}, error
+	}
+	string_data := string(data)
+	lines := strings.Split(string_data, "\n")
+	vals := strings.Split(lines[0], " ")
+	var avg [10]float64
+	sum := 0
+	for i, x := range vals {
+		if i < 2 {
```

Comment:
Why `i < 2`?
A comment explaining that might help?

Reply:
Sure. It's simply because there's an extra space between CPU and <usr> on the first line of the file. I'll add a comment